docs/guides/admin/docs/javascript/extra.js

```diff
@@ -50,12 +50,30 @@ function addTitleToCodeTag() {
     SPAN.classList.add("etc-span");
     SPAN.title = '"etc" represents the configuration directory of Opencast.\nThis directory is often located at "/etc/opencast".';
 
+    let branch = 'develop';
+    if (window.location.host == 'docs.opencast.org') {
+        branch = window.location.pathname.substring(1).replace(/\/admin.*/, '');
+    }
+    const repobase = `https://github.com/opencast/opencast/blob/${branch}/`;
+
     var codeElementList = document.getElementsByTagName("CODE");
     for (var i = 0; i < codeElementList.length; i++) {
         var CODE = codeElementList[i];
+
         if (typeof CODE.innerText !== 'undefined') {
             if (CODE.innerText.startsWith('etc/')) {
                 CODE.innerHTML = CODE.innerHTML.replace(/^etc/, SPAN.outerHTML);
+
+                // Link repository
+                let a = document.createElement('a');
+                a.innerText = '⇲';
```

Comment:
Instead of the rotated icon ⇲ (aka South East Arrow to Corner `&#8690;`), one could use 📝 (`&#128221;`) or 📄 (`&#128196;`).

Reply:
I feel like those aren't actually any better though.  At least to my eyes, the 128221 reads as 'edit this' and the other is just a gray blob.

Reply:
In that case, a gearbox should be better ⚙️, but I like the arrow shown in the example.  The magnify glass also should work 🔍.